### PR TITLE
Add ZZZ space

### PR DESCRIPTION
**Pull request by Tranquil-Flow**
Add space for ...

Etherscan: https://etherscan.io/address/0xc75f15ada581219c95485c578e124df3985e4ce0
CoinGecko: https://www.coingecko.com/en/coins/zzz-finance
Website: https://zzz.finance/
Twitter: https://twitter.com/zzzfinance

### CODE REVIEW

File: spaces/zzz-finance/index.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "token": "0xc75f15ada581219c95485c578e124df3985e4ce0",
+  "name": "ZZZ.Finance",
+  "network": "1",
+  "symbol": "ZZZ",
+  "skin": "example",
+  "domain": "https://zzz.finance/",
```
Comment (gokaiorg) [inline spaces/zzz-finance/index.json:7]: Use the domain field to add a subdomain if you have one or you should delete this field.
```suggestion
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Your images directory name is wrong. You should replace `spaces/ZZZ.finance/logo.png` and `spaces/ZZZ.finance/space.png` by ` spaces/zzz-finance/logo.png` and `spaces/zzz-finance/space.png`.

File: spaces/zzz-finance/index.json (hunk 1 above)
Comment (gokaiorg) [reply spaces/zzz-finance/index.json:7]: You should also delete the `"skin": "example",` field if you use the default skin.

**Comment by Tranquil-Flow** [comment]
@gokaiorg, made the changes as requested. Let me know if there is any further issues!

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**